src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -219,6 +220,7 @@ private void SetIds(DatabaseTopology topology, string shardedDatabaseId)
 
         public SubscriptionStorage SubscriptionStorage { get; }
 
+        public string ShardedDatabaseName { get; }
```

Comment:
why do we need this?

Reply:
in case `DocumentDatabase` is a shard, then this property will hold the Database name (without the $), for `Northwind$1` it will be `Northwind`